DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -31,21 +31,24 @@ class TTTrack : public TTTrack_TrackWord {
   double theRInv;
   double thePhi;
   double theTanL;
-  double theEta;
   double theD0;
   double theZ0;
   unsigned int thePhiSector;
+  unsigned int theEtaSector;
   double theStubPtConsistency;
   double theChi2;
-  unsigned int NumFitPars;
+  double theChi2XY;
+  double theChi2Z;
+  unsigned int numFitPars;
```

Comment:
to fit the style used in this class, should be named `theNumFitPars`
(to fit CMS style, all variables should be changed to have a trailing underscore...)

Reply:
done.